.github/workflows/sync-rest-api-docs.yaml

```
@@ -0,0 +1,49 @@
+name: Sync Camunda REST API
+
+on:
+  schedule:
+    - cron: "0 15 * * 4" # Runs every Thursday at 3 PM UTC
+  workflow_dispatch:
+
+jobs:
+  sync-file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Camunda docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda-docs
+          path: docs
+
+      - name: Checkout Camunda repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda
+          path: rest-spec
+          sparse-checkout: zeebe/gateway-protocol/src/main/proto/rest-api.yaml
+          sparse-checkout-cone-mode: false
+
+      - name: Sync documentation
+        run: |
+          cd docs
+          mv ../rest-spec/zeebe/gateway-protocol/src/main/proto/rest-api.yaml api/camunda/camunda-openapi.yaml
+          npm i
+          npm run api:generate:camunda
+          if git diff-index --quiet HEAD; then
+            echo "No changes to commit."
+            exit 0
+          fi
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          path: docs
+          title: Update camunda REST API doc
+          body: This PR updates the camunda REST API
+          commit-message: "docs: update REST API spec"
+          branch: update-rest-doc
+          delete-branch: true
+          reviewers: camunda/tech-writers
```

```suggestion
          reviewers: camunda/tech-writers,pepopowitz
```

Blocking: [The docs](https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-inputs) say this should work; I haven't tested to confirm. I would also like to be assigned  about these PRs.

---

+1 here, you should also be added as a reviewer

---

> This is why I think that the PRs coming from this workflow should be approved and merged straight forward

I appreciate your optimism, but I don't believe that mistakes won't slip through, especially as this process is forming. Yes, the specs should arrive here having already been reviewed, but (a) technical writers haven't even agreed yet to review at that stage, (b) as far as I'm aware there's nothing enforcing reviews from the devex team on those upstream spec changes, and (c) we won't know about spec changes that we aren't tagged on. 

If we see these come through a bunch of times with no missed descriptions or technical writing edits, I might feel more confident merging them blindly. But especially at the beginning, I don't share that optimism. Please assign myself and the technical writers for now.